python/pyspark/sql/window.py

```diff
@@ -117,7 +144,8 @@ def rowsBetween(start, end):
     def rangeBetween(start, end):
         """
         Creates a :class:`WindowSpec` with the frame boundaries defined,
-        from `start` (inclusive) to `end` (inclusive).
+        from `start` (inclusive) to `end` (inclusive).Range Between considers values
```

Comment:
The new text needs fixing -- missing space. But what does this mean that it 'considers values'?